lib/stripe/core_resources/customer.ex

```diff
@@ -91,7 +91,8 @@ defmodule Stripe.Customer do
                  optional(:metadata) => Stripe.Types.metadata(),
                  optional(:shipping) => Stripe.Types.shipping(),
                  optional(:source) => Stripe.id() | Stripe.Source.t(),
-                 optional(:tax_info) => Stripe.Types.tax_info()
+                 optional(:tax_info) => Stripe.Types.tax_info(),
+                 optional(:payment_method) => String.t()
```

Comment:
Awesome and thank you!  Two other spots we would need keys.  Since it is a part of the struct, we need to add it to `defstruct` and the module definition above so it isn't an unknown key ☝️

Reply:
This should do it, I think 🤔  https://github.com/code-corps/stripity_stripe/pull/569/commits/657ac009407b865772e47d764d58c4ce1554c237